week-5/C-promises/exercise.js

```diff
@@ -57,11 +63,12 @@ function exercise4() {
   Take the result of your promise and insert the it into the #exercise5
   element
 */
-
-function exercise5() {
+const exe5Promise = async () => "Hello Promises!";
```

Comment:
😄 I actually didn't think about making a Promise this way! Nice work 👍 Just be a little careful with the `async`/`await` keywords as they aren't quite fully supported yet.

There is another way of creating a Promise (the way I was thinking of!). Do you know what that would look like?

Reply:
Thanks for letting me know. In that case, we need to run our code via babel and then deploy for production build. Is that correct? I was reading what you asked us from the JS chapter 11 and tried to use async/ await.

Reply:
const exe5Promise = () => 
  new Promise((resolve, reject) => resolve("Hello Promises!"));

Reply:
Yes, you're right - Babel will allow you to convert async/await so that you can run it in production (there is a small little issue, but it's not really important).

I was mostly just making sure that you knew the other way. And yes that is indeed correct 🎉